pages/_app.tsx

```diff
@@ -12,22 +12,29 @@ import {
   LocationProvider,
   ToastProvider,
 } from '../hooks'
+import ViewContext from '../hooks/useView'
 import {
   Footer,
   Header,
   LangSwitcher,
   Toast,
   IsThisUsefulTag,
+  MobileAppBar,
 } from '../components'
 import { checkAndSetUserLocation, googlePageviews } from '../helpers/'
+import viewReducer from '../helpers/view'
 
 import '../styles/globals.css'
 import '../styles/variables.css'
 
 const App = ({ Component, pageProps }: AppProps) => {
+  const [state, dispatch] = useReducer(viewReducer, {
```

Comment:
I think you did this to help with the double loading map issue, correct?

I don't think it needs to be at the top level of the app and loaded with every page.